From 0b825c5fcf969f20615616ede23664198ed4f870 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Tue, 12 Nov 2024 22:25:31 +0100
Subject: [PATCH 1/2] tests/unit/CMakeLists.txt: Find qmltestrunner also
 without subdir

On some distributions like Alpine Linux the binary can be found in
/usr/lib/qt5/bin. Add it to the search path.
---
 tests/unit/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/unit/CMakeLists.txt b/tests/unit/CMakeLists.txt
index 0ce22019..cd96767d 100644
--- a/tests/unit/CMakeLists.txt
+++ b/tests/unit/CMakeLists.txt
@@ -1,6 +1,6 @@
 find_program(QMLTESTRUNNER_BIN
     NAMES qmltestrunner
-    PATHS /usr/lib/*/qt5/bin
+    PATHS /usr/lib/qt5/bin /usr/lib/*/qt5/bin
 )
 
 if(NOT QMLTESTRUNNER_BIN)
-- 
2.47.0

